driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -334,14 +333,50 @@ public static Bson lookup(@Nullable final String from, final List<? extends Bson
      * @param as            the name of the new array field to add to the input documents.
      * @return the $lookup pipeline stage
      * @mongodb.driver.manual reference/operator/aggregation/lookup/ $lookup
-     * @mongodb.server.release 3.6
```

Comment:
This API signature hasn't changed? So the server release and since should stay the same

Reply:
Good eye, I think I meant to paste this into the other lookup.